installing/installing_rhv/installing-rhv-customizations.adoc

```diff
@@ -7,7 +7,7 @@ toc::[]
 
 [WARNING]
 ====
-Installing {product-title} version 4.6 on {rh-virtualization-first} requires {rh-virtualization} version 4.4. If you are running an earlier version of {product-title} on {rh-virtualization} 4.3, do not update it to {product-title} version 4.6. Red Hat has not tested running {product-title} version 4.6 on {rh-virtualization} version 4.3 and does not support this combination. For more information, see link:https://access.redhat.com/articles/4763741[{product-title} 4.x Tested Integrations (for x86_x64)].
+Installing {product-title} version 4.7 on {rh-virtualization-first} requires {rh-virtualization} version 4.4. If you are running an earlier version of {product-title} on {rh-virtualization} 4.3, do not update it to {product-title} version 4.7. Red Hat has not tested running {product-title} version 4.7 on {rh-virtualization} version 4.3 and does not support this combination. For more information, see link:https://access.redhat.com/articles/4763741[{product-title} 4.x Tested Integrations (for x86_x64)].
```

Comment:
For any others that may suggest caution for this update, I double checked this statement, as it originally came from [BZ#1862586](https://bugzilla.redhat.com/show_bug.cgi?id=1862586). According to the updated support matrix, RHV 4.3 will not be supported for OCP 4.7 either. So this change is good to go. :slightly_smiling_face: